pallets/pool-fees/src/lib.rs

```diff
@@ -211,6 +211,7 @@ pub mod pallet {
 	///
 	/// Lifetime of a storage entry: Forever, inherited from pool lifetime.
 	#[pallet::storage]
+	#[pallet::getter(fn portfolio_valuation)]
```

Comment:
FYI, this will be deprecated in future polkadot versions. We should use: `pallet_loans::PortfolioValuation::<T>::get()` instead.

https://github.com/paritytech/polkadot-sdk/issues/223

Reply:
We can remove it then IMO.